tests/test_client.py

```diff
@@ -319,3 +319,14 @@ async def test_not_retry_server_errors(aiohttp_client):
         assert test_app.counter == 1
 
     await retry_client.close()
+
+
+async def test_list_retry_works_for_multiple_attempts(aiohttp_client):
+    retry_options = ListRetry(timeouts=[0]*3)
+    retry_client, test_app = await get_retry_client_and_test_app_for_test(aiohttp_client)
+    
+    async with retry_client.get('/internal_error', retry_options) as response:
+        assert response.status == 500
+        assert test_app.counter == 3
+
+    await retry_client.close()
```

Comment:
It's tested in others tests too
The thing is there is no test that check correctness of timeout selection
For example if you specify timeouts like [1, 2, 3] than the current lib will select [2, 3, raise exc], which is incorrect
I'm strongly think this is the test what we needed - check that RetryClient select retires correctly between tries

Reply:
cool, then I leave it here :+1: